core/src/components/input/test/shape/input.e2e.ts

```diff
@@ -0,0 +1,111 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('input: shape'), () => {
+    /**
+     * Solid fill is only available in MD theme.
+     */
+    test.describe('fill solid', () => {
```

Comment:
```suggestion
    test.describe('solid fill', () => {
```

Nit pick to match the other tests order